x/staking/client/cli/query.go

```diff
@@ -77,7 +77,12 @@ $ %s query staking validator cosmosvaloper1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhff
 				return fmt.Errorf("no validator found with address %s", addr)
 			}
 
-			return cliCtx.PrintOutput(types.MustUnmarshalValidator(cdc, res))
+			validator, err := types.UnmarshalValidator(types.ModuleCdc, res)
```

Comment:
Is the idea here to more generally move away from panicking? I think that makes sense esp for CLI

Reply:
Absolutely, we should never panic in client code. There are remnants of `Must*` and I've been migrating that as I go.

I would totally appreciate it if you do the same in your PRs 👍

Reply:
👍